examples/typescript/handler.ts

```diff
@@ -0,0 +1,16 @@
+export {handle};
+
+function handle(event: Record<string, unknown>, context: Record<string, unknown>, cb: unknown) {
```

Comment:
out of curiosity, is it possible to use the ts types from the offline testing package ?

Reply:
Yeah absolutely, great idea!